src/TimeModeling/Modeling/utils.jl

```diff
@@ -1,10 +1,28 @@
-
-
-function judipmap(func, iter::UnitRange; )
+function judipmap(func, iter::UnitRange; on_error = nothing)
     # Ignore map and workers if single source
     length(iter) == 1 && (return [func(iter[1])])
-    # Switch to asyncmap if serial julia
-    length(default_worker_pool()) < 2 && (return asyncmap(func, iter))
     # Standard pmap if parallel and multiple sources
-    return pmap(func, iter)
+    return pmap(func, iter; distributed = (length(default_worker_pool()) >= 2), on_error=on_error)
```

Comment:
I don't think that's necessary, `pmap` should figure out the pool size by itself.